### PR TITLE
readme: Add more information for flashing

### PR DESCRIPTION
**Pull request by crawfxrd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,38 @@ source ~/.cargo/env
...
+2. Remove the bottom panel
+3. Attach the clip to the SPI ROM chip
+4. Connect to the Raspberry Pi and wait it for boot
+```
+sudo tio -b 115200 /dev/ttyUSB0
+```
+5. From the RPi session, test that the chip is detected.
+```
+cd firmware
+./spipi.sh -r backup.rom
```
Comment (brs17) [inline README.md]: This is all in a private repository where as this repository is open. Therefore I am not sure this content belongs here.

Comment (crawfxrd) [reply README.md]: Are you referring to this file specifically? `./scripts/spipi-*.sh` already reference this file (this being a simplified `spipi-dump.sh` to test the SPI clip connection), and it's contents are nothing worth keeping secret.

Comment (brs17) [reply README.md]: This is good with open sourcing the spipi script and all related code.

**Comment by crawfxrd** [comment]
I've changed the address to be a default for now, so users can override it by exporting `SPIPI` before running the script.

**Pull request merged**